source/github_ee/github_ee_test.go

```diff
@@ -0,0 +1,36 @@
+package github_ee
+
+import (
+	"net/http"
+	"net/http/httptest"
+	nurl "net/url"
+	"strings"
+	"testing"
+)
+
+func Test(t *testing.T) {
```

Comment:
Because the API path for EE is `host.com/api/v3` and `github.com` doesn't resolve the api at `github.com/api/v3`, it's hard for me to test anything else, other than verifying that the enterprise API path was being fetched.

Since EE driver uses default Github driver, the migration tests are covered by that.

Reply:
I think using a test server is fine.